ballerina/observe.bal

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/observe;
+
+const REPORTER_NAME = "newrelic";
+const PROVIDER_NAME = "newrelic";
+
+configurable string apiKey = "";
+
+function init() {
+    if (apiKey == "") {
+        io:println("error: cannot find API key for trace API. Please configure API key in Config.toml file.");
```

Comment:
Shall we throw an error instead of `io:print` here? As we can't proceed without the key, the application will exit with the error.

Reply:
Addressed the review suggestions